src/hdx_cli/cli_interface/dictionary/commands.py

```diff
@@ -46,23 +47,28 @@ def files(ctx: click.Context):
                'usercontext': profileinfo}
 
 
-@click.command(help='Create dictionary.')
-@click.argument('dictionary_description_file')
+@click.command(help=f"Create dictionary. 'dictionary_file' contains the settings. "
+                    "The filename and name in settings will be replaced by 'dictionary_filename' "
+                    "and 'dictionary_name' respectively.")
```

Comment:
This is confusing. We already have something called dictionary file. It's the file that gets uploaded separately that a dictionary points at. And then when creating the dictionary we have a different file just for the dictionary settings, but we also call it dictionary file?

Reply:
Regarding this, yes.. you are right. 
Does it make more sense if I rename "dictionary_file" to "dictionary_settings"?